pkg/api/documentsschema/documentsschema.proto

```diff
@@ -0,0 +1,252 @@
+syntax = "proto3";
+
+package immudb.documentsschema;
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/struct.proto";
+import "schema.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+option go_package = "github.com/codenotary/immudb/pkg/api/documentsschema";
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+  base_path: "/api/v2",
+  info: {
+    title: "immudb REST API v2";
+    description: "Objects storage API"
+  };
+  security_definitions: {
+    security: {
+      key: "bearer"
+      value: {
+        type: TYPE_API_KEY
+        in: IN_HEADER
+        name: "grpc-metadata-sessionid"
+        description: "Session token: <token>"
+      }
+    }
+  }
+  security: {
+    security_requirement: {
+      key: "bearer"
+    }
+  }
+};
+
+message DocumentInsertRequest {
+  string collection = 1;
+  repeated google.protobuf.Struct document = 2;
```

Comment:
just to be aware of: it won't allow multi-collection insertions

Reply:
I think we will not allow that